src/server/submit/confirmation/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Sets up the routes used in the submit confirmation page.
+ * These routes are registered in src/server/router.js.
+ */
+
+import { submitConfirmationController } from './controller.js'
+
+/**
+ * @satisfies {ServerRegisterPluginObject<void>}
+ */
+export const submitConfirmation = {
+  plugin: {
+    name: 'confirmation',
+    register(server) {
+      server.route([
+        {
+          method: 'GET',
+          path: '/submit/confirmation',
+          ...submitConfirmationController
+        }
+      ])
+    }
+  }
+}
```

Comment:
We've got a generic page controller for this now - that'll generate the plugin for us. See `src/server/destination/general-licence/index.js` for an example